### ISSUE TITLE
Welcome_AI4D: Alt text is provided to the decorative image which is not correct

### ISSUE THREAD

**Issue by fsteffi** [wixtoolset/issues#6798]
GithubTags:#A11yMAS;#A11YTCS;#WCAG1.1.1;#A11ySev2;#AI4D;#DesktopApp;#Win10;

## **Environment Details:**

Application Name: PowerShell Core [Benchmark]
 

Application Version: Powershell-7.0.0-winx64.msi

Windows Version: Windows10

 
## **Repro Steps: ​​**

1. Launch PowerShell Core application.
2. Click on PowerShell application file-> next>>next>> next>> next>> Installation.
3. Start Accessibility Inside for Desktop.

## **Actual**:

Alt text is provided to the decorative image which is not corect.
 
## **Expected**:

Alt text should not be provided to the decorative image.

## **User Impact:**

Screen Reader users will not get correct indications because of Descriptive  image name.

## **Attachment**
![image](https://user-images.githubusercontent.com/24580570/172325198-2ea96152-c12c-4bef-98ed-48e329cfc4c4.png)

**Comment by barnson** [comment]
Windows Installer owns that control. WiX cannot change how it handles accessibility.
